andela/travel_tool_back
# #160196032 Approve/Reject Request (By Requester's Manager)

#### What does this PR do?
- requesters manager should only be able to update a request's status once

#### Description of Task to be completed?
- add code block in `updateApprovals` method to check if a request has been updated by an approver
- stop the update process if the above is true

How should this be manually tested?
- Clone the repo -git clone https://github.com/andela/travel_tool_back.git
- Enter into the project directory - cd travel_tool_back
- Start the app using make start
- Login on your browser and grab your token from the application tab in Developers tab
- Open Postman
- Add your token in headers
- Send a PUT request to http://localhost:5000/api/v1/approvals/:requestId with newStatus in req.body

Any background context you want to provide?
- Create a new request first on `My Requests` page and ensure to add yourself as manager on the form.

#### What are the relevant pivotal tracker stories?
[#160196032](https://www.pivotaltracker.com/story/show/160196032)

#### Screenshots (if appropriate)
<img width="1080" alt="screen shot 2018-09-09 at 10 04 42 pm" src="https://user-images.githubusercontent.com/23608060/45268936-7f54ac00-b47c-11e8-9aca-6a4f12a5e1da.png">

#### Questions:
